packages/react-core/src/components/TextInputGroup/TextInputGroupMain.tsx

```diff
@@ -53,6 +53,8 @@ export interface TextInputGroupMainProps extends Omit<React.HTMLProps<HTMLDivEle
   isExpanded?: boolean;
   /** @beta The id of the element(s) controlled by the input. Required if role has a value of "combobox". */
   'aria-controls'?: string;
+  /** The id of the input element */
+  id?: string;
```

Comment:
More of a nit, but maybe `inputId` instead, that way an `id` can still be spread to the wrapper div element.

Reply:
Good point! Updated 🙂